scripts/release/release-new-branch.sh

```diff
@@ -67,18 +67,23 @@ git fetch upstream \
   || error_exit "Could not fetch upstream."
 git branch ${RELEASE_BRANCH} ${SHA} \
   || error_exit "Could not create a local release branch."
-git push upstream ${SHA}:refs/heads/${RELEASE_BRANCH} \
+git push upstream -f ${SHA}:refs/heads/${RELEASE_BRANCH} \
```

Comment:
isn't `-f` a little risky? It can overwrite the branch. Why does it need to overwrite, aren't we just adding new commits on top?

Reply:
Done.